<ui version="4.0" >
 <author>Johannes Simon</author>
 <class>ChartConfigWidget</class>
 <widget class="QWidget" name="ChartConfigWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>434</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item rowspan="2" row="0" column="0" >
    <widget class="QPushButton" name="chartTypeMenu" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="layoutDirection" >
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff/>
      </iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QCheckBox" name="threeDLook" >
     <property name="text" >
      <string>3D</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QPushButton" name="editData" >
     <property name="text" >
      <string>Edit Data...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="pushButton_5" >
     <property name="text" >
      <string>Themes...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>282</width>
        <height>318</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Plot Area</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="10" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>61</width>
           <height>71</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="5" >
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QCheckBox" name="showTitle" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Title</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="showSubTitle" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Subtitle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="showFooter" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Footer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="5" >
        <widget class="QPushButton" name="pushButton_4" >
         <property name="text" >
          <string>Background...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="5" >
        <widget class="Line" name="line_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt; font-weight:600;">Axes&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3" >
        <widget class="QComboBox" name="axes" />
       </item>
       <item row="9" column="0" colspan="5" >
        <widget class="QPushButton" name="axisScalingButton" >
         <property name="text" >
          <string>Scaling...</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3" >
        <widget class="QToolButton" name="addAxis" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="4" >
        <widget class="QToolButton" name="removeAxis" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="5" >
        <widget class="Line" name="line_3" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-weight:600;">Labels&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="4" >
        <widget class="QLineEdit" name="axisTitle" />
       </item>
       <item row="7" column="0" >
        <widget class="QCheckBox" name="axisShowTitle" >
         <property name="text" >
          <string>Label</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="5" >
        <widget class="QCheckBox" name="axisShowGridLines" >
         <property name="text" >
          <string>Show Grid Lines</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>282</width>
        <height>318</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Data Sets</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="3" >
        <widget class="QComboBox" name="dataSets" >
         <property name="toolTip" >
          <string>Select a data set</string>
         </property>
         <property name="statusTip" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2" >
        <widget class="KColorButton" name="datasetColor" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Attached Axis</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2" >
        <widget class="QComboBox" name="dataSetAxes" >
         <property name="sizeAdjustPolicy" >
          <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
         </property>
         <item>
          <property name="text" >
           <string>sdf</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0" colspan="3" >
        <widget class="Line" name="propertiesSeparator" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3" >
        <widget class="QLabel" name="barPropertiesLabel" >
         <property name="text" >
          <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt; font-weight:600;">Bar Properties&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2" >
        <widget class="QLabel" name="gapBetweenBarsLabel" >
         <property name="text" >
          <string>Gap Between Bars</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2" >
        <widget class="QSpinBox" name="gapBetweenBars" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="suffix" >
          <string>%</string>
         </property>
         <property name="maximum" >
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2" >
        <widget class="QLabel" name="gapBetweenSetsLabel" >
         <property name="text" >
          <string>Gap Between Sets</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2" >
        <widget class="QSpinBox" name="gapBetweenSets" >
         <property name="suffix" >
          <string>%</string>
         </property>
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="value" >
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="9" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>31</width>
           <height>94</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1" colspan="2" >
        <widget class="QPushButton" name="dataSetChartTypeMenu" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="styleSheet" >
          <string notr="true" >text-align:left;</string>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="autoRepeat" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="datasetColorLabel" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3" >
        <layout class="QHBoxLayout" name="horizontalLayout_2" >
         <item>
          <widget class="QCheckBox" name="datasetShowValues" >
           <property name="text" >
            <string>Data Values</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="dataSetShowLabels" >
           <property name="text" >
            <string>Labels</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" >
        <widget class="QCheckBox" name="dataSetHasChartType" >
         <property name="text" >
          <string>Chart Type</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>282</width>
        <height>318</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Legend</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="3" >
        <widget class="QCheckBox" name="showLegend" >
         <property name="text" >
          <string>Show Legend</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4" >
        <widget class="Line" name="line" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_8" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3" >
        <widget class="QLineEdit" name="legendTitle" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2" >
        <widget class="QCheckBox" name="legendShowFrame" >
         <property name="text" >
          <string>Show Frame</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Orientation</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" >
        <widget class="QRadioButton" name="legendOrientationIsVertical" >
         <property name="text" >
          <string>v</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="3" >
        <widget class="QRadioButton" name="legendOrientationIsHorizontal" >
         <property name="layoutDirection" >
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text" >
          <string>h</string>
         </property>
         <property name="icon" >
          <iconset>
           <normaloff/>
          </iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="4" >
        <widget class="QPushButton" name="pushButton_3" >
         <property name="text" >
          <string>Position and Alignment...</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>101</width>
           <height>121</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
